src/helpers/xml-builder.js

```diff
@@ -590,6 +590,17 @@ const buildIndentation = (left = 720) => {
   return indentationFragment;
 };
 
+const buildHeading = (heading = 'Heading1') => {
```

Comment:
essentially we are building paragraphStyle or pStyle with this method, and I would suggest the appropriate name for it would be buildPStyle and hence associated naming should be named as pStyleFragment etc.